### PR TITLE
Create download_page.md

### PR DESCRIPTION
**Pull request by diamonddocumentation**
Download page draft version 1

### CODE REVIEW

**Comment by diamonddocumentation** [comment]
This is the first draft of download page for comments/review

**Comment by oscarisalittledog** [comment]
Ok I think I get this now.

File: download_page.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+1. [Download the latest version ZAP 2.7.0](https://github.com/zaproxy/zaproxy/wiki/Downloads) here
+2. Launch the installation wizard by double clicking on the downloaded executable file ![GitHub Logo](/images/logo.png)
+3. Read the License agreement and Click 'Accept' to continue the installation
+4. Select 'Standard' or 'Custom' installation
+5. Click 'Finish' to exit set up ![Finish dialog image]
+
+### Starting Up
+
+1. Launch ZAP ![image]
+2. Select 'Yes, I want to Persist Sessions...' to save ZAP sessions for later retrieval ![image]
```
Comment (oscarisalittledog) [inline download_page.md]: Don't like this please change

File: download_page.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## Download ZAP
+
+ZAP needs to be installed on the system you intend to perform pentesting on.
+
+### Installation
+1. [Download the latest version ZAP 2.7.0](https://github.com/zaproxy/zaproxy/wiki/Downloads) here
```
Comment (oscarisalittledog) [inline download_page.md]: This is great

File: download_page.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Select the Quick Start tab of the Workspace Window ![workspace image]
+2. Enter the full URL of the web application you want to attack. [test website](http://www.webscantest.com/) This website is intentionally vulnerable and safe to test
+3. Right click '.....' to open the URL and proxy via ZAP in the browser of your choice ![image]
+4. Click the Attack button.
+
+#### How to Interpret your Test Results
+
+As ZAP spiders your web application, it constructs a map of your web applications’ pages
+and the resources used to render those pages. Then it records the requests and responses
```
Comment (oscarisalittledog) [inline download_page.md:31]: This is really well written

**Comment by diamonddocumentation** [comment]
Please comment on these changes

**Pull request closed without merging**